packages/clay-color-picker/.eslintrc

```diff
@@ -0,0 +1,9 @@
+{
+	"plugins": ["react"],
+	"rules": {
+		"no-unused-vars": ["error", {"ignoreRestSiblings": true}],
+		"react/jsx-uses-react": "error",
+		"react/jsx-uses-vars": "error",
+		"no-mixed-spaces-and-tabs": ["error", "smart-tabs"]
```

Comment:
We might want to add `react/jsx-sort-props` here (perhaps the "plugin:react/recommended" preset includes it already, and using the preset would probably be preferable); it looks like you've generally tried to keep things in alphabetical order, but there are some places where things are out of order.